src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs

```diff
@@ -133,6 +135,9 @@ public Size Viewport
             private set { SetAndRaise(ViewportProperty, ref _viewport, value); }
         }
 
+        /// <inheritdoc/>
+        IControl IScrollAnchorProvider.CurrentAnchor => _anchor.Item1;
```

Comment:
I think you could just write `_anchor.control` for clarity. Since in some places you are using `Item1` and in some `control`.